File: src/test/java/no/nav/bidrag/aktoerregister/service/TSSServiceMock.java
package no.nav.bidrag.aktoerregister.service;

import no.nav.bidrag.aktoerregister.persistence.entities.Aktoer;

public class TSSServiceMock implements AktoerService {

  @Override
  public Aktoer hentAktoer(String aktoerIdent) {
    return Aktoer.builder()
        .aktoerIdent(aktoerIdent)
        .aktoerType("AKTOERNUMMER")
        .adresselinje1("Testgate 1")
        .build();
  }
}

File: src/test/kotlin/no/nav/bidrag/aktoerregister/service/TSSServiceMock.kt
package no.nav.bidrag.aktoerregister.service

import no.nav.bidrag.aktoerregister.persistence.entities.Aktør

class TSSServiceMock : AktoerService { //TODO() Slette?
    override fun hentAktoer(aktoerIdent: String): Aktør {
        return Aktør(
            aktørIdent = aktoerIdent,
            aktørType = "AKTOERNUMMER",
            adresselinje1 = "Testgate 1"
            )
    }
}